install_config/install/advanced_install.adoc

```diff
@@ -617,6 +620,26 @@ value; you only need to set this if you want your `git clone` operations to use
 a different value.
 |===
 
+[[advanced-install-configuring-firewalls]]
+=== Configuring the Firewall
+
+{product-title} uses iptables as the default firewall, but you can switch to use firewalld in the install.
+
+Because iptables is the default filewall, {product-title} is designed to have it
```

Comment:
Careful here... firewalld is the default recommended firewall in RHEL.  How about:

"Because `iptables` the previous default firewall, {product-title} still sets it as the default so adding a node to an existing cluster doesn't cause inconsistencies."